x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/detail_panel/detail_panel.js

```diff
@@ -0,0 +1,404 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+
+import {
+  EuiButton,
+  EuiCallOut,
+  EuiDescriptionList,
+  EuiDescriptionListDescription,
+  EuiDescriptionListTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiIcon,
+  EuiLoadingSpinner,
+  EuiSpacer,
+  EuiText,
+  EuiTextColor,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { CRUD_APP_BASE_PATH } from '../../../constants';
+
+import { ConnectionStatus, RemoveClusterButton } from '../components';
+
+export class DetailPanelUi extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    isLoading: PropTypes.bool,
+    cluster: PropTypes.object,
+    closeDetailPanel: PropTypes.func.isRequired,
+    clusterName: PropTypes.string,
+    copyCluster: PropTypes.func.isRequired,
+  }
+
+  copyCluster = () => {
+    const { copyCluster, cluster } = this.props;
+    const {
+      name,
+      seeds,
+      skipUnavailable,
+    } = cluster;
+
+    const clusterConfig = {
+      name,
+      seeds,
+      skipUnavailable,
+    };
+
+    copyCluster(clusterConfig);
+  }
+
+  renderSkipUnavailableValue(skipUnavailable) {
+    if(skipUnavailable === true) {
+      return (
+        <FormattedMessage
+          id="xpack.remoteClusters.detailPanel.skipUnavailableTrueValue"
+          defaultMessage="Yes"
+        />
+      );
+    }
+
+    if(skipUnavailable === false) {
+      return (
+        <FormattedMessage
+          id="xpack.remoteClusters.detailPanel.skipUnavailableFalseValue"
+          defaultMessage="No"
+        />
+      );
+    }
+
+    return (
+      <FormattedMessage
+        id="xpack.remoteClusters.detailPanel.skipUnavailableNullValue"
```

Comment:
__optional nit:__ maybe `Default` instead of `Null` as `Null` sounds like implementation detail.